docs/rag_guide.md

```
@@ -0,0 +1,236 @@
+        db_path: <path-to-vector-index> # e.g. /home/USER/vector_db/faiss_store.db
+        namespace: null
+
+vector_dbs:
+- embedding_dimension: <embedding-dimension> # e.g. 768
+  embedding_model: <embedding-model-name> # e.g. sentence-transformers/all-mpnet-base-v2
+  provider_id: custom-index
+  vector_db_id: <index-id> 
+```
+
+Where:
+- `provider_model_id` is the path to the folder of the embedding model (or alternatively, the supported embedding model to download)
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)